src/tutorial/runtime/introduction.rst

```diff
@@ -291,60 +334,163 @@ Where Do All The Files Go?
 
          ~/cylc-run/<workflow-name>/log/job/<cycle-point>/<task-name>/<job-submission-num>/
 
-      .. image:: ../img/cylc-gui-view-log.png
-         :align: center
-         :scale: 75%
+      .. TODO REPLACE THIS IF APPROPRIATE
+
+         .. image:: ../img/cylc-gui-view-log.png
+            :align: center
+            :scale: 75%
 
 
-Running A Workflow
-------------------
+Installing A Workflow
+---------------------
 
 .. ifnotslides::
 
-   It is a good idea to check a workflow for errors before running it.
-   Cylc provides a command which automatically checks for any obvious
-   configuration issues called ``cylc validate``, run via:
+   .. seealso::
 
-.. code-block:: sub
+      :ref:`The full guide to Cylc install <Installing-workflows>`.
+
+   .. versionchanged:: 8.0.0
+
+      ``cylc install`` is a new feature of Cylc 8.
+
+   To allow you to separate the development and running of workflows
+   Cylc provides a :term:`cylc install <install>` command.
+
+For a workflow developed in
 
-   cylc validate <path/to/workflow>
+- ``~/cylc-src/my_workflow`` and
+- ``/some/location/my_other_workflow``
+
+respectively:
+
+.. code-block:: bash
+
+   cylc install my_workflow
+   cylc install -C /some/location/my_other_workflow
 
 .. ifnotslides::
 
-   Here ``<path/to/workflow>`` is the path to the workflow's location within the
-   filesystem (so if we create a workflow in ``~/cylc-run/foo`` we would put
-   ``~/cylc-run/foo/flow.cylc``).
+   will install your workflow in ``~/cylc-run/my_workflow/runN``.
 
-   Next we can run the workflow using the ``cylc play`` command.
+   (RunN is actually a symlink to runX where X increments each time you
+   install a workflow)
```

Comment:
I think this will be confusing to people new to `runN`. Probably leave it out of this section?
```suggestion
```